src/rabbitmonitor.js

```diff
@@ -37,6 +37,39 @@ class RabbitMonitor {
     this.queuePrefix = '/queue/' + namespacePrefix;
   }
 
+  // useful code for deleting connections, queues, exchanges for a namespace - copied here
+  // when this functionality was removed from the API
```

Comment:
We can probably just delete it and keep a reference to the commit where it used to be or put this code in a bug somewhere. Either way is fine with me